test/dtest/suite.yaml

```diff
@@ -5,3 +5,6 @@ cluster:
 extra_scylla_config_options:
   authenticator: AllowAllAuthenticator
   authorizer: AllowAllAuthorizer
+run_in_dev:
```

Comment:
It sounds like you want **all** dtests to run only in dev mode, so maybe we need a way to put here a wildcard - all files in this directory will be run_in_dev?

Reply:
Unfortunately, we have no wildcard for these lists in `test.py`